lr-cli/cortx_setup/commands/resource/discover.py

```diff
@@ -15,13 +15,47 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 
-
+import time
 from ..command import Command
+from provisioner.commands import PillarSet
+from cortx.utils.discovery import Discovery
 
 
 class ResourceDiscover(Command):
+
+    def get_resource_map(self, resource_type='nodes[0]'):
+        retries = 30
+        status = ""
+        self.logger.debug("Running resource discover command")
+        # Currently since this is a bug once fixed
+        # we wont be needing to provide any parameter
+        request_id = int(Discovery.generate_node_health(resource_type))
```

Comment:
Do we have one to one mapping of Provisioner related resource type to SSPL related resource type?

Reply:
Also lets log message with generated request ID?

Reply:
Is `Discovery` a static class or a class method or has initialization been missed?
static would be weird, but can't help if it is.

Reply:
@ypise Yes it is static.
@nitin-seagate  Done